src/scripts/dimApp.module.js

```diff
@@ -35,17 +39,20 @@ import run from './dimApp.run';
 import state from './state';
 import loadingTracker from './services/dimLoadingTracker.factory';
 
+// required to make ng-i18next work
+window.i18next = i18next;
```

Comment:
TODO: I actually know how to do this slightly more elegantly with webpack's `alias` now. No big deal though.